### PR TITLE
Added example of using OpenSSL to generate keys in the JWT RBAC docs

### LINKED ISSUE #8202: Update JWT guide with how to generate a key pair with openssl
**Description**
Jan Kunzmann has posted this snippet:

```
openssl genrsa -out private_key.pem 4096
openssl rsa -pubout -in private_key.pem -out public_key.pem
# convert private key to pkcs8 format in order to import it from Java
openssl pkcs8 -topk8 -in private_key.pem -inform pem -out private_key_pkcs8.pem -outform pem -nocrypt
```
Updating the guide with it would help the users to start faster with signing/verifying tokens

**Implementation ideas**

I will check it locally and update the docs

### PR DESCRIPTION
**Pull request by nderwin**
Fixes #8202

* added a section for generating the keys using openssl using the provided example commands as a guide
* updated the section about using the private key to reference back to the new section for generating the keys

Signed-off-by:Nathan Erwin <nathan.d.erwin@gmail.com>

### CODE REVIEW

**Comment by gastaldi** [comment]
Can you change the PR title to match the intent? 

See https://github.com/quarkusio/quarkus/blob/master/COMMITTERS.adoc#pull-request-title

**Comment by nderwin** [comment]
> Can you change the PR title to match the intent?
> 
> See https://github.com/quarkusio/quarkus/blob/master/COMMITTERS.adoc#pull-request-title

Absolutely, and thanks for the link, I'll remember that for next time!

File: docs/src/main/asciidoc/security-jwt.adoc
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-jwt.adoc:336]: @nderwin Thanks for this update, IMHO it would be better to keep it not as one of the main subsections, but as a `Note`, `Generating Keys with openssl`, as generating the keys openssl is not necessarily what the users will generally do and have this note immediately under the private key pem example, instead of injecting `(or use the privateKey.pem file from <<Generating Keys>>)` (otherwise we;ll also need to do the same for the public key).
So have this note and suggest the end - that `you can use the generated pair of keys instead of the keys used by this demo` or something like that...

Does it sound reasonable ?

File: docs/src/main/asciidoc/security-jwt.adoc
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-jwt.adoc:343]: Please use `2048` as the keys used in the demo are `2048` bit length AFAIK as well (if you'd like, please add a comment that one can increase this value to `4096`/etc)

File: docs/src/main/asciidoc/security-jwt.adoc
Comment (nderwin) [reply docs/src/main/asciidoc/security-jwt.adoc:336]: Sure!  I'll make those changes and update the PR shortly.

File: docs/src/main/asciidoc/security-jwt.adoc
Comment (nderwin) [reply docs/src/main/asciidoc/security-jwt.adoc:343]: Oops, I didn't notice that.  I'll update it.

**Comment by gastaldi** [comment]
Nice! Can you please squash your commits? See https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md#coding-guidelines for more details :wink:

**Comment by nderwin** [comment]
> Nice! Can you please squash your commits? See https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md#coding-guidelines for more details

Alright, let's see if I did that right...

**Comment by sberyozkin** [comment]
@nderwin Thanks, it all looks good, I don't recall your original commit message, but can you please do: `git commit --amend -m 'Example of using OpenSSL to generate keys in the JWT RBAC docs'` and force push again ?

**Review by sberyozkin — APPROVED** [review]

**Comment by nderwin** [comment]
> @nderwin Thanks, it all looks good, I don't recall your original commit message, but can you please do: `git commit --amend -m 'Example of using OpenSSL to generate keys in the JWT RBAC docs'` and force push again ?

Better?

**Comment by gastaldi** [comment]
I think a better commit message would be: 

```
Example of using OpenSSL to generate keys in the JWT RBAC docs 

Fixes #8202
```

**Comment by nderwin** [comment]
> I think a better commit message would be:
> 
> ```
> Example of using OpenSSL to generate keys in the JWT RBAC docs 
> 
> Fixes #8202
> ```

Ok - do you want me to kill the sign-off as well, or should that remain?  I checked the contributing doc, but it didn't call that out specifically.

**Comment by gastaldi** [comment]
The sign-off isn't required, it's up to you ;)

**Comment by gastaldi** [comment]
Thank you!

**Comment by nderwin** [comment]
> Thank you!

No, no, thank _you_!  :grin:

**Comment by sberyozkin** [comment]
@nderwin thank you :-)

**Pull request merged**
